standalone/src/ExRootTreeWriter.h

```diff
@@ -0,0 +1,52 @@
+#ifndef ExRootTreeWriter_h
+#define ExRootTreeWriter_h
+
+/** \class ExRootTreeWriter
+ *
+ *  Class handling output ROOT tree
+ *
+ *  \author P. Demin - UCL, Louvain-la-Neuve
+ *
+ */
+
+#include "TNamed.h"
+
+#include <set>
+
+class TFile;
+class TTree;
+class TClass;
+class ExRootTreeBranch;
+
+class ExRootTreeWriter: public TNamed
+{
+public:
+  ExRootTreeWriter(TFile *file = 0, const char *treeName = "Analysis");
+  ~ExRootTreeWriter();
+
+  void SetTreeFile(TFile *file) { fFile = file; }
+  void SetTreeName(const char *name) { fTreeName = name; }
+
+  ExRootTreeBranch *NewBranch(const char *name, TClass *cl);
+
+  void SetTree(TTree* t) { fTree = t; }
```

Comment:
Add a comment to indicate the changes wrt upstream?

Reply:
Done.